lib/sparql.ts

```diff
@@ -490,6 +496,8 @@ function translateProject(op: Algebra.Project | Algebra.Ask | Algebra.Describe,
 
 function objectContainsValues(o: any, vals: string[]): boolean
 {
+    if (Util.isTerm(o))
+        return vals.indexOf(translateTerm(o)) >= 0;
```

Comment:
termToString is not used anywhere. stringToTerm is though so since we already use the library rdf-string we might as well use termToString as well instead of our own function :) good point!

Reply:
Done! Good suggestion.